Mathlib/GroupTheory/QuotientGroup/Defs.lean

```diff
@@ -243,6 +243,20 @@ theorem lift_quot_mk {φ : G →* M} (HN : N ≤ φ.ker) (g : G) :
     lift N φ HN (Quot.mk _ g : Q) = φ g :=
   rfl
 
+@[to_additive]
+theorem lift_surjective_of_surjective (φ : G →* M) (hφ : Function.Surjective φ) (HN : N ≤ φ.ker) :
+    Function.Surjective (QuotientGroup.lift N φ HN) :=
+  fun x ↦ ⟨mk (hφ x).choose, by rw [lift_mk, (hφ x).choose_spec]⟩
+
+@[to_additive]
+theorem ker_lift (φ : G →* M) (HN : N ≤ φ.ker) :
+    (QuotientGroup.lift N φ HN).ker = Subgroup.map (QuotientGroup.mk' N) φ.ker := by
```

Comment:
See also `Subgroup.map_comap_eq_self_of_surjective` + `MonoidHom.comap_ker` + (missing) `(QuotientGroup.lift N φ HN).comp (QuotientGroup.mk' N) = φ := rfl`

Reply:
I added a new proof as you suggested. It is a bit convoluted since it has to through `Submonoid`.